frameworks/digital-outcomes-and-specialists/questions/briefs/specialistRole.yml

```diff
@@ -76,6 +81,11 @@ options:
       Develop and deliver an effective user-focused digital service.
       Manage the full product lifecycle, including user research, design, delivery and continuous improvement.
   -
+    label: Technical Architect
```

Comment:
Sorry to be a pedant, but someone other than me (Roz in all likelihood) might complain about the capitalisation of `Architect` here, as all other roles only have the first word capitalised.